Makefile

```diff
@@ -854,24 +895,33 @@ $(BUILD_DIR)/assets/audio/samplebank_table.h: $(SAMPLEBANK_BUILD_XMLS)
 $(BUILD_DIR)/assets/audio/soundfont_table.h: $(SOUNDFONT_BUILD_XMLS) $(SAMPLEBANK_BUILD_XMLS)
 	$(ATBLGEN) --fonts $@ $(SOUNDFONT_BUILD_XMLS)
 
+SEQ_ORDER_DEFS := -DDEFINE_SEQUENCE_PTR\(name,seqId,_2,_3,_4\)=*\(name,seqId\) \
+                  -DDEFINE_SEQUENCE\(name,seqId,_2,_3,_4\)=\(name,seqId\)
```

Comment:
ngl I find this pretty ugly. Is it really so much harder to parse the header directly? Say with more gratuitous use of `strchr()`, and maybe a helper function to trim whitespace

Reply:
I did this to reduce the chances of the parsing failing depending on what a user might do to the file. I think it's much safer than trying to approximately parse the contents of the full header.

Reply:
hm maybe it's a mistake to have these tables be in a C header if things besides C need to use them. Seems like that ship has sailed though

Reply:
Perhaps not better, but what about just having a `sequence_order.in` in assets that can add the defines and include the table?
Something like <https://github.com/zeldaret/mm/commit/5d5db31fc2a1da38c16d29bca9a2b014e019facd>